tools/clang/include/clang/SPIRV/FeatureManager.h

```diff
@@ -19,11 +19,10 @@
 
 #include "clang/Basic/Diagnostic.h"
 #include "clang/Basic/SourceLocation.h"
+#include "clang/Frontend/CompilerInstance.h"
```

Comment:
Do we need the CompilerInstance header? I think `SPIRVOptions.h` should be fine?

Reply:
good point. Done